services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -101,38 +101,23 @@ func DefaultConfig() *config.Config {
 				Apps: []string{"files", "search", "text-editor", "pdf-viewer", "external", "admin-settings", "epub-reader"},
 				ExternalApps: []config.ExternalApp{
 					{
-						ID:   "preview",
-						Path: "web-app-preview",
-						Config: map[string]interface{}{
-							"mimeTypes": []string{
-								"image/tiff",
-								"image/bmp",
-								"image/x-ms-bmp",
-							},
-						},
+						ID:     "preview",
```

Comment:
Now with the mime types config being removed you can remove the preview app from the external apps entirely and add `preview` to the `Apps` slice in line 101. We only had it as external app to be able to pass config.